azurerm/resource_arm_subnet.go

```diff
@@ -66,6 +66,39 @@ func resourceArmSubnet() *schema.Resource {
 				Optional: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
+
+			"delegation": {
```

Comment:
it appears we're missing a flatten function to set these back into the state?

Reply:
done